http/vibe/http/common.d

```diff
@@ -712,6 +712,45 @@ final class Cookie {
 		}
 	}
 
+	/// Parses the cookie from a header field, returning the name of the cookie.
+	string parse(string headerString) {
+		auto parts = headerString.splitter("; ");
```

Comment:
That would be except the first one, thus making the first one require different treatment. So I think it may be better to leave this as is, and throw the exception if any part contains ';', though that would be a little slower, as you have to walk the string instead of just comparing the first char.

Reply:
That plus `indexOf`` with a string argument is also considerably slower than the overload taking a single character. But isn't the first part already handled separately outside of the loop anyway?

Reply:
I've been doing some looking, and according to the RFC ([https://tools.ietf.org/html/rfc6265#section-4.1.1](url)) for Set-Cookie, a space after the semi-colon is not required, and all leading / trailing spaces should be stripped. I'll probably switch to a `splitter(';')` and use `trim(key)` and `trim(value)`